mlflow/tracking/client.py

```diff
@@ -694,6 +694,44 @@ def end_trace(
 
         self.end_span(request_id, root_span_id, outputs, attributes, status, end_time_ns)
 
+    def _log_trace(self, trace: Trace) -> str:
```

Comment:
Can we mark this as experimental since it is technically part of a main public-facing API obj (even though it IS private, I do believe users will find this easily and it should be marked as subject to modification / deletion if a different approach is decided on).

Reply:
Yup makes sense:)